Detections/MultipleDataSources/SucessfullPosswordSprayAttampOnAWSConsolelogin.yaml

```diff
@@ -0,0 +1,121 @@
+id: 188db479-d50a-4a9c-a041-644bae347d1f
+name: Successful Password Spray Attempt on AWS Console login

```

Comment:
I would rename this to better reflect the activity - maybe something like "Successful AWS Console Login from IP Address Observed Conducting Password Spray"

Reply:
changed